.cicd/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. ./$HELPERS_DIR/execute.sh
+
+execute mkdir -p $ROOT_DIR/build
+
+if [[ $(uname) == Darwin ]]; then
+    MAC_CMAKE="cmake -DCMAKE_BUILD_TYPE=Release .."
+    MAC_MAKE="make -j$JOBS"
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
+    cd $ROOT_DIR
+    ccache -s
+    cd build
+    if [[ $BUILDKITE ]]; then
+        if [[ $ENABLE_BUILD ]]; then
+            execute $MAC_CMAKE
+            execute $MAC_MAKE
+        elif [[ $ENABLE_TEST ]]; then
+            execute $MAC_TEST
+        fi
+    elif [[ $TRAVIS ]]; then
+        execute mkdir -p wasms
+        execute $MAC_CMAKE
+        execute $MAC_MAKE
+        #execute $MAC_TEST
+    fi
```

Comment:
This entire block of code...
```
if [[ $BUILDKITE ]]; then
        if [[ $ENABLE_BUILD ]]; then
            execute $MAC_CMAKE
            execute $MAC_MAKE
        elif [[ $ENABLE_TEST ]]; then
            execute $MAC_TEST
        fi
    elif [[ $TRAVIS ]]; then
        execute mkdir -p wasms
        execute $MAC_CMAKE
        execute $MAC_MAKE
        #execute $MAC_TEST
    fi
```
...can be simplified as follows:
```
[[ "$TRAVIS" == 'true' ]] && execute mkdir wasms
execute cmake -DCMAKE_BUILD_TYPE=Release ..
execute make -j$JOBS
[[ "$ENABLE_TEST" == 'true' ]] && ctest -j$JOBS --output-on-failure -T Test
```
If it needs to be more complicated than this, perhaps it would be best to have separate build and test scripts.